.github/workflows/auto_community_labeler.yml

```diff
@@ -0,0 +1,27 @@
+
+name: Auto Community Labeler
+
+on:
+  pull_request:
+  issues:
+
+jobs:
+  auto_PR_labeler:
+    name: Auto PR Community Labeler
+    runs-on: ubuntu-latest
+    if: ${{ github.event_name == 'pull_request' && github.event.pull_request.author_association != 'MEMBER' && github.event.pull_request.author_association != 'OWNER' }}
+    steps:
+      - name: Add Contributor Label
+        uses: actions-ecosystem/action-add-labels@v1
+        with:
+          labels: community contribution
+
+  auto_Issue_labeler:
+    name: Auto Issue Community Labeler
+    runs-on: ubuntu-latest
+    if: ${{ github.event_name == 'issues' && github.event.issue.author_association != 'MEMBER' && github.event.issue.author_association != 'OWNER' }}
+    steps:
+      - name: Add Contributor Label
+        uses: actions-ecosystem/action-add-labels@v1
+        with:
+          labels: community contribution
```

Comment:
```suggestion
name: Auto Community Labeler

# only run on initial issue or pull request open
on:
  issues:
    types:
      - opened
      
  pull_request:
    types:
      - opened

jobs:
  auto_community_labeler:
    name: Auto PR Community Labeler
    runs-on: ubuntu-latest
    # run if the author is both not a MEMBER and not a OWNER
    if: |
      (github.event_name == 'pull_request' && github.event.pull_request.author_association != 'MEMBER' && github.event.pull_request.author_association != 'OWNER') ||
      (github.event_name == 'issues' && github.event.issue.author_association != 'MEMBER' && github.event.pull_request.author_association != 'OWNER')
    steps:
      - name: Add Contributor Label
        uses: actions-ecosystem/action-add-labels@v1
        with:
          labels: community contribution

```

I wonder if this would work.